tests/lib/rules/no-wait-for-snapshot.test.ts

```diff
@@ -51,73 +51,101 @@ ruleTester.run(RULE_NAME, rule, {
       `,
     })),
     ...ASYNC_UTILS.map((asyncUtil) => ({
+      settings: {
+        'testing-library/utils-module': 'test-utils',
+      },
       code: `
         import { ${asyncUtil} } from 'some-other-library';
-        test('snapshot calls within ${asyncUtil} are not valid', async () => {
+        test('aggressive reporting disabled - snapshot calls within ${asyncUtil} not related to Testing Library are valid', async () => {
```

Comment:
After checking if coming from Testing Library directly inside `isAsyncUtil`, this test was invalid. I opted-out aggressive reporting so this test keeps passing, and moved the original one to invalid test cases.